common/faults/fault_config.json

```diff
@@ -651,7 +651,7 @@
                 },
                 {
                     "fault_name": "pack_temp_exceeded",
-                    "max": 6000,
+                    "max": 600,
                     "min": 0,
                     "priority": "error",
                     "time_to_latch": 2000,
```

Comment:
maybe make this 500ms, 2s seems kinda long

Reply:
fixed